package.json

```diff
@@ -17,6 +17,8 @@
     "diagrams": "mermaid --sequenceConfig guides/assets/esa-initial-flow.config.json --width 2400 -o guides/assets guides/assets/esa-initial-flow.txt",
     "fastboot": "echo \"Please also run npm start\"; ember fastboot --serve-assets",
     "lint": "eslint app addon blueprints config server test-support tests *.js",
+    "now-build": "ember build --environment=production",
```

Comment:
I wonder whether we should actually not build production so people can inspect the app's source easily. Or we could deploy sourcemaps as well.